api/authentication/backends.py

```diff
@@ -34,6 +33,7 @@ def authenticate(self, request):
                 return None
 
             token = authorization_header.split(' ')[1]
+            print(f"Token= {token}")
```

Comment:
Do a self review, remove all residual comments and print statements before asking for review.